packages/docs/docs/getting-started.md

```diff
@@ -2,223 +2,62 @@
-    Jalara was always willing to lend her magic to aid those in need, but she was also fiercely independent, and many feared her wrath.
-    \n\nHer greatest triumph came when she defeated the dark wizard, Zoltar, in a fierce magical battle.
-    The victory cemented her place as one of the most powerful sorceresses in the land.
-    Though Jalara no longer seeks out adventure in the way she once did, her wisdom and magic continue to shape the fate of the Kingdom of Arcadia.
-    Born in the Kingdom of Eldor, Aria is a powerful sorceress who possesses an exceptional talent for magic.
-    She discovered her gift at a young age and spent years honing her skills, studying under some of the most renowned wizards in the land.
-    With her mystical powers, she has become a valuable asset to King Eramis, who often seeks her prophetic advice on matters concerning the realm.
-    Despite her great power and wisdom, Aria remains humble and compassionate, always looking out for the welfare of her people.
-    Her most notable achievement to date was orchestrating the warding spell that saved Eldor from the infamous Night Dragon, which had long terrorized the kingdom.
-    Aria is revered by many and feared by her enemies, who know all too well the consequences of crossing her."
-  }
-],
+$ git clone https://github.com/fixie-ai/ai-jsx
+$ cd ai-jsx
+$ yarn
+$ yarn build
 ```
 
-## Beyond Text: Image Generation
-
-You are not restricted to text models. Let's create an image for the story as well:
-
-```tsx title="index.tsx"
-function WriteStoryWithImage() {
-  const story = memo(<WriteStory />);
-  return (
-    <>
-      Banner URL:{' '}
-      <ImageGen>
-        <ChatCompletion>
-          <UserMessage>
-            You are an artist. You are creating a sketch for a new book. Concisely describe a sketch for scene from the
-            following story: {story}
-          </UserMessage>
-        </ChatCompletion>
-      </ImageGen>
-      {'\n\n'}
-      {story}
-    </>
-  );
-}
-```
+You will find examples in the directories `packages/examples`, `packages/tutorial`,
+`packages/create-react-app-demo`, and `packages/nextjs-demo`.
 
-We use a second LLM pass to create a description for the image. We then feed this description to an `<ImageGen>` component.
-`<ImageGen>` by default uses [Dalle](https://platform.openai.com/docs/guides/images/introduction).
+You can then run the various examples from the top level of the `ai-jsx` checkout using
+`yarn workspace`, like so:
 
-Note that we also used a new component, `memo`. If we simply use `<WriteStory />` twice (even if you assign it to a variable), you will have two separate LLM calls and possibly two different stories as a result.
-For reference see [Memoization](guides/rules-of-jsx.md#Memoization).
+```
+$ yarn workspace tutorial run part1
+$ yarn workspace examples run demo:debate
+$ yarn workspace create-react-app-demo start
```

Comment:
The preferred command, as noted in [the AI-UI page](https://docs.ai-jsx.com/guides/ai-ui), is:

```
yarn turbo run dev --scope create-react-app-demo
```